cmd/skuba/cluster/status.go

```diff
@@ -32,7 +33,12 @@ func NewStatusCmd() *cobra.Command {
 		Use:   "status",
 		Short: "Show cluster status",
 		Run: func(cmd *cobra.Command, args []string) {
-			if err := cluster.Status(); err != nil {
+			clientSet, err := clientset.GetAdminClientSet()
+			if err != nil {
+				klog.Errorf("Can not retrieve the ClientSet from kubernetes: %v", err)
+			}
```

Comment:
[1]

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md
I would rather to change remove.go to make it consistent.